SPECS/aws-nitro-enclaves-cli.spec

```diff
@@ -130,18 +130,24 @@ echo -e "
 "
 
 %preun
-# Uninstall services
-%systemd_preun nitro-enclaves-vsock-proxy.service
-%systemd_preun nitro-enclaves-allocator.service
+if [ $1 -ne 1 ]; then
+    # Any operation except for package upgrade
+    # Uninstall services
+    %systemd_preun nitro-enclaves-vsock-proxy.service
+    %systemd_preun nitro-enclaves-allocator.service
```

Comment:
Aren't these macros already handling the checks themselves ?

Reply:
Indeed, this is the case. Will remove the redundant check.